brainfuck/doc.go

```diff
@@ -0,0 +1,8 @@
+/*
+Package brainfuck contains functionality to interpret any program
+written in Brainfuck language.
+Usage:
+Put a file with program in Brainfuck language into directory "examples",
+then run it from CLI
```

Comment:
`then run it from CLI`. Pls provide an example here. I can not understand how to do that.

Reply:
you need to use command: `go run ./. examples/example.bf`, also description was updated